.travis.yml

```diff
@@ -42,29 +41,25 @@ jobs:
     - name: "Build documentation"
       env: >-
         PYTHON_VERSION=3.6
-      python: "3.6"
       stage: pre-test
       script: ./scripts/ci/ci_docs.sh
     - name: "Tests postgres python 3.6"
       env: >-
         BACKEND=postgres
         ENV=docker
         PYTHON_VERSION=3.6
-      python: "3.6"
       stage: test
     - name: "Tests sqlite python 3.6"
       env:
         BACKEND=sqlite
         ENV=docker
         PYTHON_VERSION=3.6
-      python: "3.6"
       stage: test
     - name: "Tests mysql python 3.7"
       env:
         BACKEND=mysql
         ENV=docker
         PYTHON_VERSION=3.7
-      python: "3.7"
```

Comment:
So this is the only major difference? - that this will run on a python 3.6 host in a python 3.7 docker env?

Reply:
Exactly. This make a difference for example for static checks which do not use container (for example debug statements check). It's not relevant for the: code tests/mypy/pylint/flake (they are all using containers with PYTHON_VERSION. 

BTW. See here for all the tests with "Breeze" * -> those all use PYTHON_VERSION and containers:  https://github.com/apache/airflow/blob/master/CONTRIBUTING.rst#supported-pre-commit-hooks